docs/kivy.md

```diff
@@ -2,6 +2,11 @@
 
 Must have Cython installed before installing Kivy
 - Cython version 0.28.5
+- Kivy version 1.10.1
 
-To run the application using python3:
-` kivy3 main.py` from inside the directory that contains the file
+To set everything up:
+- Activate the virtual environment .bin/venv/activate
+- ` pip install Cyton==0.28.5`
```

Comment:
can't these go into the requirements.txt file?

Reply:
Can we just add a second requirements.txt then? One for pre-kivy stuff?
Then people can just do pip install x.txt, pip install y.txt, and travis can do the same

Reply:
It's an edit to tox.ini, which travis will use automatically.
At:
https://github.com/dbgrigsby/Postr/blob/master/tox.ini 
Where it says 
```
[testenv]
deps = -r{toxinidir}/requirements.txt
```
Just change the line to
```
[testenv]
deps =
    -r{toxinidir}/requirements_kivy.txt
    -r{toxinidir}/requirements.txt
```